cms/models.py

```diff
@@ -348,6 +348,27 @@ class FacultyMembersPage(CourseProgramChildPage):
     ]
 
 
+class ImageCarouselPage(CourseProgramChildPage):
+    """
+    Page that holds image carousel.
+    """
+
+    header = models.CharField(
```

Comment:
Probably more straightforward to use the existing `title` field inherited from `Page`